internal/osbuild2/yum_config_stage.go

```diff
@@ -0,0 +1,45 @@
+package osbuild2
+
+import "fmt"
+
+type YumConfigConfig struct {
+	HttpCaching string `json:"http_caching,omitempty"`
+}
+
+type YumConfigPlugins struct {
+	Langpacks YumConfigPluginsLangpacks `json:"langpacks,omitempty"`
+}
+
+type YumConfigPluginsLangpacks struct {
+	Locales []string `json:"locales"`
```

Comment:
`locales` must not be empty (https://github.com/osbuild/osbuild/blob/40bc2161274abac895ce74a323fada30674b3d2f/stages/org.osbuild.yum.config#L45). Feel free to disregard if you considered it, but I wonder whether it could / should be tested in `YumConfigStageOptions`'s `validate()` method.

Reply:
Sure, do you think it is ok to just check for length > 0 or would you like to see a regex matching those values as well (in such case I'd change the schema as well)?